### PR TITLE
Xfail test_ntp and test_ntp_long_jump_disabled due to not supported

### PR DESCRIPTION
**Pull request by wsycqyz**
### Description of PR

Summary:
From 202311 branch, the test_ntp and test_ntp_long_jump_disabled are not supported. So xfail them.
The xfail condidtion mark is in 202311 branch but not in master and 202405 branch.
Add them back.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

**Comment by ZhaohuiS** [comment]
test_ntp and test_ntp_long_jump_disabled is not consistently failed, about 50% pass rate, why sometimes it passed but sometimes it failed? what makes it flaky?
If it's due to https://github.com/sonic-net/sonic-buildimage/issues/19425,
Can we add the iburst option  in ntp.conf while running ntp cases? Also need to restart ntp-config and ntp services.

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -1373,6 +1373,21 @@ nat:
       - "'nat' not in feature_status"
 
 #######################################
+#####           ntp               #####
+#######################################
+ntp/test_ntp.py::test_ntp:
+  xfail:
+    reason: "iburst not enabled by default on 202311 and newer images"
```
Comment (ZhaohuiS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1380]: should we add version condition here?

**Comment by wsycqyz** [comment]
> test_ntp and test_ntp_long_jump_disabled is not consistently failed, about 50% pass rate, why sometimes it passed but sometimes it failed? what makes it flaky? If it's due to [sonic-net/sonic-buildimage#19425](https://github.com/sonic-net/sonic-buildimage/issues/19425), Can we add the iburst option in ntp.conf while running ntp cases? Also need to restart ntp-config and ntp services.

In 202311 and 202405, the iburst is already there.
image

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 1 above)
Comment (wsycqyz) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1380]: Will add version here.

**Comment by wsycqyz** [comment]
> test_ntp and test_ntp_long_jump_disabled is not consistently failed, about 50% pass rate, why sometimes it passed but sometimes it failed? what makes it flaky? If it's due to [sonic-net/sonic-buildimage#19425](https://github.com/sonic-net/sonic-buildimage/issues/19425), Can we add the iburst option in ntp.conf while running ntp cases? Also need to restart ntp-config and ntp services.

Find the flaky reason: IPv6 is not working for some subnets in the lab.

**Pull request closed without merging**